tools/bakeddata-scripts/template/Cargo.toml.template

```diff
@@ -21,4 +21,8 @@ cldr = { tagged = "_cldr_tag_" }
 icuexport = { tagged = "_icuexport_tag_" }
 segmenter_lstm = { tagged = "_segmenter_lstm_tag_" }
 
+[lints.rust]
+unexpected_cfgs = { level = "warn", check-cfg = ['cfg(icu4x_custom_data)'] }
```

Comment:
Question: Why level = warn?

Reply:
Oh I copied this from the error message. We don't need it.